src/MainWindow.vala

```diff
@@ -212,8 +212,12 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
         };
         headerbar.set_custom_title (search_clamp);
         headerbar.pack_start (return_button);
-        headerbar.pack_end (menu_button);
-        headerbar.pack_end (view_mode_revealer);
+        if (!Utils.is_running_in_guest_session ()) {
+            headerbar.pack_end (menu_button);
+            headerbar.pack_end (view_mode_revealer);
+        } else {
+            headerbar.pack_end (new Gtk.Label (""));
```

Comment:
If I do not call `headerbar.pack_end`, the headerbar is broken. Is there a better way than adding an empty widget?